ui/package.json

```diff
@@ -86,10 +84,8 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
```

Comment:
Don't we need node-sass for building css etc?

Reply:
I found that removing it has no effect on the build process. I'm not sure about this, can keep it if we want.

Reply:
@utchoang I suppose it's worth checking where/how it is used, esp for any `less` and related stylesheet processing. (it's possible it doesn't happen or is required for build, but maybe for development)